tests/foreman/cli/test_sso.py

```diff
@@ -19,7 +19,7 @@ class TestSSOCLI(CLITestCase):
     # can be easily added later.
 
     @stubbed()
-    def test_sso_kerberos_cli(self):
+    def test_positive_login_kerberos(self):
```

Comment:
I think we should append `user` in test name. Like `test_positive_login_kerberos_user` because all different tests are using different auth users. Just a suggestion.

Same comment for rest of the tests.